WeakAurasOptions/AuthorOptions.lua

```diff
@@ -704,6 +705,178 @@ local typeControlAdders = {
+            local childOption = childData.authorOptions[optionID]
+            childOption.default[k] = v
+            WeakAuras.Add(childData)
+          end
+          WeakAuras.ReloadTriggerOptions(data[0])
+        else
+          option.default[k] = v
+          WeakAuras.Add(data)
+          WeakAuras.ReloadTriggerOptions(data)
+        end
+      end,
+    }
+    order = order + 1
+    for j, value in ipairs(values) do
+      args["option" .. i .. "space" .. j] = {
+        type = "toggle",
+        width = WeakAuras.normalWidth,
+        name = L["Value %i"]:format(j),
+        order = order,
+        disabled = function() return true end,
+        get = function() return true end,
+        set = function() end,
+      }
+      order = order + 1
+      args["option" .. i .. "value" .. j] = {
+        type = "input",
+        width = WeakAuras.normalWidth - 0.15,
+        name = (value == conflict and "|cFF4080FF" or "") .. L["Value %i"]:format(j),
+        desc = descSelect(data, option, j, conflict),
+        order = order,
+        get = function()
+          if value ~= conflict then
+            return value
+          end
+        end,
+        set = function(_, value)
+          if option.references then
+            for childID, optionID in pairs(option.references) do
+              local childData = data[childID]
+              local childOption = childData.authorOptions[optionID]
+              local insertPoint = math.min(j, #childOption.values + 1)
+              if value == "" then
+                tremove(childOption.values, insertPoint)
+              else
+                childOption.values[insertPoint] = value
+              end
+              WeakAuras.Add(childData)
+            end
+            WeakAuras.ReloadTriggerOptions(data[0])
+          else
+            if value == "" then
+              tremove(values, j)
+            else
+              values[j] = value
+            end
+            WeakAuras.Add(data)
+            WeakAuras.ReloadTriggerOptions(data)
+          end
+        end
+      }
+      order = order + 1
+      args["option" .. i .. "valdelete" .. j] = {
+        type = "execute",
+        width = 0.15,
+        name = "",
+        order = order,
+        func = function()
+          if option.references then
+            for childID, optionID in pairs(option.references) do
+              local childData = data[childID]
+              local childOption = childData.authorOptions[optionID]
+              tremove(childOption.values, j)
+              WeakAuras.Add(childData)
+            end
+            WeakAuras.ReloadTriggerOptions(data[0])
+          else
+            tremove(values, j)
+            WeakAuras.Add(data)
+            WeakAuras.ReloadTriggerOptions(data)
+          end
+        end,
+        image = "Interface\\AddOns\\WeakAuras\\Media\\Textures\\delete",
+        imageWidth = 24,
+        imageHeight = 24
+      }
+    end
+    args["option" .. i .. "newvaluespace"] = {
```

Comment:
Not really a space :)

Reply:
This follows the convention in other parts of the codebase, where a disabled checkbox on the left is called a space. Also it’s the same as the select code above.